README.md

```diff
@@ -19,217 +19,170 @@
-- `top`:
-	- [`block-io`](docs/builtin-gadgets/top/block-io.md)
-	- [`ebpf`](docs/builtin-gadgets/top/ebpf.md)
-	- [`file`](docs/builtin-gadgets/top/file.md)
-	- [`tcp`](docs/builtin-gadgets/top/tcp.md)
-- `trace`:
-	- [`bind`](docs/builtin-gadgets/trace/bind.md)
-	- [`capabilities`](docs/builtin-gadgets/trace/capabilities.md)
-	- [`dns`](docs/builtin-gadgets/trace/dns.md)
-	- [`exec`](docs/builtin-gadgets/trace/exec.md)
-	- [`fsslower`](docs/builtin-gadgets/trace/fsslower.md)
-	- [`mount`](docs/builtin-gadgets/trace/mount.md)
-	- [`oomkill`](docs/builtin-gadgets/trace/oomkill.md)
-	- [`open`](docs/builtin-gadgets/trace/open.md)
-	- [`signal`](docs/builtin-gadgets/trace/signal.md)
-	- [`sni`](docs/builtin-gadgets/trace/sni.md)
-	- [`tcp`](docs/builtin-gadgets/trace/tcp.md)
-	- [`tcpconnect`](docs/builtin-gadgets/trace/tcpconnect.md)
-	- [`tcpdrop`](docs/builtin-gadgets/trace/tcpdrop.md)
-	- [`tcpretrans`](docs/builtin-gadgets/trace/tcpretrans.md)
-- [`traceloop`](docs/builtin-gadgets/traceloop.md)
+Inspektor Gadget is an [eBPF](https://ebpf.io/) framework and set of tools to debug and inspect
+containers, Kubernetes clusters and Linux hosts. It manages the packaging, deployment and
+execution of eBPF programs encapsulated in [OCI images](https://opencontainers.org/) called Gadgets.
 
-## Installation
+TODO: Insert terminal/promo video
+
+## Features
+
+* Build and package eBPF programs into OCI images called Gadgets
+* Targets Kubernetes clusters and Linux hosts
+* Collect and export data to observability tools with a simple command (and soon via declarative configuration)
+* Security mechanisms to restrict and lock-down which Gadgets can be run
+* Automatic [enrichment](#what-is-enrichment): map kernel data to high-level resources like Kubernetes and container runtimes
+* Supports [WebAssembly](https://webassembly.org/) modules to post-process data and customize IG [operators](#what-is-an-operator); using any WASM-supported language 
+* Supports many modes of operation; cli, client-server, API, embeddable via Golang library
+
+## Quick start
+
+This assumes you have the required Inspektor Gadget tools installed as outlined in the [Installation instructions](#installation).
+
+The following examples use the [trace_open Gadget](/gadgets/trace_open/README.md) which triggers when a file is open on the system.
 
-Install Inspektor Gadget (client-side):
+### Kubernetes 
 
-Use [krew](https://sigs.k8s.io/krew) plugin manager to install:
+Deploy and run on a Kubernetes cluster
 
 ```bash
-$ kubectl krew install gadget
+kubectl gadget deploy
+kubectl gadget run trace_open
 ```
 
-Install Inspektor Gadget on Kubernetes:
+### Linux
+
+Runs Inspektor Gadget locally on Linux.
 
 ```bash
-$ kubectl gadget deploy
+ig run trace_open
 ```
 
-Read the detailed [install instructions](https://www.inspektor-gadget.io/docs/latest/getting-started/) to find more information.
+### MacOS or Windows (but not exclusively)
```

Comment:
I'm not sure this title is right. Should it be somethin glike "using gadgetctl"? It's also possible to use kubectl-gadget from MacOS / Windows and this part doesn't make it clear.

Reply:
I know, but that doesn't talk to folks who are wondering how to use IG on Mac or Windows. I'm assuming they don't know `gadgetctl`'s purpose and aren't looking for how to use it but how to use IG itself on those platforms.